config.xsd

```diff
@@ -21,42 +21,43 @@
             <xs:element name="globals" type="GlobalsType" minOccurs="0" maxOccurs="1" />
         </xs:choice>
 
-        <xs:attribute name="name" type="xs:string" />
-        <xs:attribute name="useDocblockTypes" type="xs:string" />
-        <xs:attribute name="useDocblockPropertyTypes" type="xs:string" />
-        <xs:attribute name="throwExceptionOnError" type="xs:string" />
-        <xs:attribute name="hideExternalErrors" type="xs:string" />
         <xs:attribute name="autoloader" type="xs:string" />
         <xs:attribute name="cacheDirectory" type="xs:string" />
-        <xs:attribute name="allowFileIncludes" type="xs:string" />
-        <xs:attribute name="totallyTyped" type="xs:string" />
-        <xs:attribute name="strictBinaryOperands" type="xs:string" />
-        <xs:attribute name="requireVoidReturnType" type="xs:string" />
-        <xs:attribute name="useAssertForType" type="xs:string" />
-        <xs:attribute name="rememberPropertyAssignmentsAfterCall" type="xs:string" />
-        <xs:attribute name="serializer" type="xs:string" />
-        <xs:attribute name="allowPhpStormGenerics" type="xs:string" />
-        <xs:attribute name="allowCoercionFromStringToClassConst" type="xs:string" />
-        <xs:attribute name="allowStringToStandInForClass" type="xs:string" />
-        <xs:attribute name="ignoreInternalFunctionFalseReturn" type="xs:string" />
-        <xs:attribute name="ignoreInternalFunctionNullReturn" type="xs:string" />
-        <xs:attribute name="usePhpDocMethodsWithoutMagicCall" type="xs:string" />
-        <xs:attribute name="memoizeMethodCallResults" type="xs:string" />
-        <xs:attribute name="hoistConstants" type="xs:string" />
-        <xs:attribute name="addParamDefaultToDocblockType" type="xs:string" />
-        <xs:attribute name="checkForThrowsDocblock" type="xs:string" />
-        <xs:attribute name="checkForThrowsInGlobalScope" type="xs:string" />
-        <xs:attribute name="forbidEcho" type="xs:string" />
         <xs:attribute name="errorBaseline" type="xs:string" />
-        <xs:attribute name="findUnusedCode" type="xs:string" />
-        <xs:attribute name="findUnusedVariablesAndParams" type="xs:string" />
-        <xs:attribute name="parseSql" type="xs:string" />
         <xs:attribute name="maxStringLength" type="xs:string" />
-        <xs:attribute name="resolveFromConfigFile" type="xs:string" />
-        <xs:attribute name="includePhpVersionsInErrorBaseline" type="xs:string" />
-        <xs:attribute name="loadXdebugStub" type="xs:string" />
-        <xs:attribute name="ensureArrayStringOffsetsExist" type="xs:string" />
-        <xs:attribute name="ensureArrayIntOffsetsExist" type="xs:string" />
+        <xs:attribute name="name" type="xs:string" />
+        <xs:attribute name="serializer" type="xs:string" />
+
+        <xs:attribute name="addParamDefaultToDocblockType" type="xs:boolean" default="false" />
```

Comment:
I've used this as a reference to set the type and default:
https://psalm.dev/docs/running_psalm/configuration/#optional-ltpsalm-gt-attributes

Some of these are not mentioned in the docs, guess the docs are lagging behind?